README.md

```diff
@@ -89,59 +92,26 @@ public void onTasksChanged() {
 
-#### Conceptual complexity
+Adding or amending features to this version of the app requires a similar amount of effort to the todo-mvp sample. Before modifying the sample, you should be familiar with the [Loaders API](https://developer.android.com/guide/components/loaders.html), which is a relatively complex topic.
 
-Developers need to be familiar with the Loaders framework, which is not
-trivial.
+This sample introduces two new classes, in addition to those found in todo-mvp: [`TaskLoader`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-loaders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TaskLoader.java), and [`TasksLoader`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-loaders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksLoader.java). Some aspects of the code are simpler as loaders take care of asynchronous work. The table below summarizes the amount of code used to implement this version of the app. You can compare it with similar tables provided for each of the other samples in this project.
 
-### Testability
-
-#### Unit testing
-
-The use of the Loaders framework adds a big dependency with the Android
-framework so unit testing is harder.
-
-#### UI testing
-
-No difference with MVP.
-
-### Code metrics
-
-Compared to MVP, the only new classes are TaskLoader and TasksLoader. Parts of
-the code are simpler as Loaders take care of the asynchronous work.
-
-
-```
--------------------------------------------------------------------------------
-Language                     files          blank        comment           code
--------------------------------------------------------------------------------
-Java                            48           1085           1444           3517 (3450 in MVP)
-XML                             34             97            337            601
--------------------------------------------------------------------------------
-SUM:                            82           1182           1781           4118
--------------------------------------------------------------------------------
-
-```
-### Maintainability
 
-#### Ease of amending or adding a feature
+| Language      | Number of files | Blank lines | Comment lines | Lines of code |
+| ------------- | --------------- | ----------- | ------------- | ------------- |
+| **Java**      |               48|         1085|           1444|           3571 (3450 in todo-mvP)|
+| **XML**       |               34|           97|            337|            601|
+| **Total**     |               82|         1182|           1781|           4118|
 
-Similar to MVP
 
-#### Learning cost
+## Comparing this sample
 
-Medium as the Loaders framework is not trivial.
+The following summary reviews how this solution compares to the todo-mvp base sample:
 
+ * <b>Use of architectural frameworks, libraries, or tools: </b>None required.
```

Comment:
Again just based on the original, but I've updated it to mention the Loaders API.